app/views/time_clock_periods/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<% breadcrumb :time_clock_periods %>
+
+<%= search_form_for @q, data: {"controller": "turbo_search", "turbo-frame": "results"} do |f| %>
+<div class="card">
+  <div class="card-header">
+    <h1>Time Clock Periods</h1>
+  </div>
+  <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <%= f.input :name_cont, label: 'Name' %>
+      </div>
+      <div class="col">
+        <%= f.input :team_id_eq, label: 'Team', as: :select, collection: policy_scope(Team) %>
+      </div>
+      <div class="col">
+        <%= f.input :start_time_gteq, label: 'From', as: :datetime %>
+      </div>
+      <div class="col">
+        <%= f.input :end_time_lteq, label: 'To', as: :datetime  %>
+      </div>
+    </div>
+  </div>
+  <div class="card-footer">
+    <%= link_to "New Time Clock Period", new_time_clock_period_path, class: 'btn btn-success' if policy(TimeClockPeriod).new? %>
+    <%= link_to "Time Clock Punches", time_clock_punches_path, class: 'btn btn-primary' %>
```

Comment:
This should be a `btn-info` or something other than `btn-primary`

Reply:
Done